recipes-nodes/rbuilder-bidding/init

```diff
@@ -11,10 +11,7 @@
 PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 NAME=rbuilder-bidding
 DESC="Builder Bidding Service"
-LOGFILE=/var/log/rbuilder-bidding.log
-LOGFILE_MONITOR=/var/log/rbuilder-bidding_monitor.log
-PIDFILE=/var/run/rbuilder-bidding.pid
-PIDFILE_MONITOR=/var/run/rbuilder-bidding_monitor.pid
+LOGFILE=/var/log/containers/rbuilder-bidding.log
```

Comment:
same here, I would make it /var/log/rbuilder-bidding.log

Reply:
see above